lib/runner.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var _path = require('path');
+
+var _path2 = _interopRequireDefault(_path);
+
+var _fs = require('fs');
+
+var _fs2 = _interopRequireDefault(_fs);
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+function _toArray(arr) { return Array.isArray(arr) ? arr : Array.from(arr); }
+
+var LGRC_FILENAME = _path2.default.join(process.env.HOME, '.lgrc');
+
+function getUserOptions() {
+  try {
+    _fs2.default.accessSync(LGRC_FILENAME, _fs2.default.R_OK); // will throw if not readable
+  } catch (error) {
+    return null;
+  }
+  var userOptions = JSON.parse(_fs2.default.readFileSync(LGRC_FILENAME).toString());
+  return userOptions;
+}
+
+function run(commandAndArgv) {
+  process.env.APP_BASEURL = 'https://game-cli.learnersguild.org';
```

Comment:
Looks like we need this for the `Origin` header for the graphql query. Is that header required by us or by graphql? Does it have to be a url? This feels a little strange but isn't hurting anything. Mostly just curious.